scripts/constants.ts

```diff
@@ -9,6 +9,7 @@ export const triBarAddress = "0x802119e4e253D5C19aA06A5d567C5a41596D6803";
 export const triMakerAddress = "0xe793455c9728fc91A3E5a33FAfF9eB2F228aE151";
 export const factoryAddress = "0xc66F594268041dB60507F00703b152492fb176E7";
 export const multiSigAddress = "0xf86119de6ee8d4447C8219eEC20E7561d09816d3";
+export const stableUsnMaker = "0x1232726DA91B25D22239C5707FE85E8F078F3532" // TODO: Replace with real stableUsnMaker
```

Comment:
what is the process here? should we wait for this PR to be merged before `stableUsnMaker` gets merged+launched or lmerge this PR?

Reply:
ya, we can merge this and then add another PR?